roles/run_awx_tests/tasks/main.yml

```diff
@@ -16,6 +16,13 @@
     docker pull gcr.io/ansible-tower-engineering/awx_devel:{{ target_branch }}
   ignore_errors: yes
 
+- name: Create docker file if necessary
+  template:
+    src: installer/roles/image_build/templates/Dockerfile.j2
```

Comment:
I think both of these need to be prepended with: `{{ source_dir }}`. See where we use that below.